analysis/analyze_ranks.m

```diff
@@ -196,7 +196,11 @@
         % F1 ... fragments (rows) x repeats (columns) of raw failure count.
         
         % Average accuracy is average over valid frames (non NaN).
-        average_accuracy(t1) = mean(O1(valid_frames));
+	if all(valid_frames == 0)
```

Comment:
What I find strange about this is that valid_frames should be a logical vector, not a scalar. If it is a scalar then this means that the sequence only has one frame?